src/views/studio/EditVideoSheet/EditVideoForm/EditVideoForm.tsx

```diff
@@ -221,7 +221,7 @@ export const EditVideoForm: React.FC<EditVideoFormProps> = ({
       setFileSelectError('Video file cannot be empty')
       return
     }
-    if ((!isEdit || dirtyFields.assets?.thumbnail) && !data.assets.thumbnail?.blob) {
+    if (!data.assets.thumbnail?.url && !dirtyFields.assets?.thumbnail) {
```

Comment:
It seems to work but I don't understand how. The condition for the error is `no URL and field not dirty`. How does that work for a new video? In that case the field shouldn't be dirty if the thumbnail wasn't provided right?

Reply:
Oh, wait. I just noticed that `dirtyFields.assets?.thumbnail` is always `undefined`, no matter what you do with it. I could just set this to `if (!data.assets.thumbnail?.url)` and I'll do that